platform/administer/templates/versioning.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Versioning Templates
+sidebar_label: Versioning
+sidebar_position: 4
+---
+
+import Flow, { Step } from "@site/src/components/Flow";
+import NavStep from "@site/src/components/NavStep";
+import Button from "@site/src/components/Button";
+import Label from "@site/src/components/Label";
+
+Templates can be versioned. Versioning a template allows you
+to make changes over time to a template while retaining the current version/iteration. Resources (for example virtual clusters, namespaces) that are deployed with templates that have updated versions show up with a warning
+indicating that a newer template version is available.
+
+## Template version strings
+
+When creating a virtual cluster from a template, either through the UI or CLI, you can specify a template version string. If omitted, the latest available version is selected by default.
+
+You can use the `X` wildcard in the major, minor, or patch components of the version string to enable automatic upgrades. This allows virtual clusters to automatically update to newer compatible versions of the template.
+
+For example given a template version `1.0.0`:
+
+- If you set the version string to `1.X.X`, and a new template version `1.1.0` is added, the virtual cluster will automatically upgrade to `1.1.0`.
+- If a version `2.0.0` is added instead, the cluster **will not** upgrade, because the major version does not match.
+
+To always upgrade to the latest available version, you can `X.X.X` as the version string.
+
+:::note
+Avoid using `X.X.X` unless necessary. Use the `X` placeholder only in the minor or patch position—for example, `1.X.X` or `1.2.X`—to maintain greater control over version updates.
+:::
+
+## Create a template version
+
+You can add new versions to your templates using the UI. The steps are the same for all resources. 
```

Comment:
You can also add a new version via a GitOps approach by directly applying (kubectl, Argo CD, .) an update `VirtualClusterInstance` to the vCluster Platform cluster.